### DISCUSSION TITLE
One Support Center Navigation after Login

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3213] [Office hours] [answered]
### Background

We are exploring some navigational options for One Support Center that aligns with the new One Twilio Theme.

### Feedback

The current Support Center experience only has 4 sections in total, so we have been exploring how that might look from a navigational perspective.  Eventually the new One Support Center will be housed in a different platform than Console, which allows greater control over the experience, but will still link back to Console.
- How closely does the navigation need to follow Console, since the experience is going to be housed on a different platform?
- Do we need to continue following the construct of a left navigation or can we potentially explore other options?

[Figma file
](https://www.figma.com/file/gqSvdYJnFoUZRwJFWPSm9i/One-Support-Center?node-id=1673-326435&t=oeLzd5GDioFbPTTt-0)

### Stage of design

Exploration

### Are you looking for visual design support?

- [X] Yes

**Comment by nkrantz** [comment, marked as answer]
Hi @tonge495 - following up with Office Hours from this past Thurs. [If you'd like to review the recording, it can be found here ](https://www.loom.com/share/2f28039a33104da2afb29f93978148f7)(password in #help-design-systems slack). Please keep us updated with your plans and constraints around navigational components! Thanks for attending.
